hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -522,15 +537,38 @@ public FileStatus getFileStatus(final Path path) throws IOException {
             eTag);
   }
 
+  /**
+   * @param path The list path.
+   * @return the entries in the path.
+   * */
   public FileStatus[] listStatus(final Path path) throws IOException {
-    LOG.debug("listStatus filesystem: {} path: {}",
+    return listStatus(path, null);
+  }
+
+  /**
+   * @param path Path the list path.
+   * @param startFrom the entry name that list results should start with.
```

Comment:
Could you give an example of `startFrom` please? From the tests it seems like the `startFrom` is supposed to be a relative name within `path`?
Should this API be called like:
```java
listStatus("/some-folder", "file-in-folder")
listStatus("/some-folder", "file-in")
```
or
```java
listStatus("/some-folder", "/some-folder/file-in-folder")
listStatus("/some-folder", "/some-folder/file-in")
```
from the test cases it seems like the former.

Reply:
Yes, the former one is correct. `startFrom` is the name of an entry,  not a full path.
I will add an example here.